examples/js/loaders/TDSLoader.js

```diff
@@ -272,6 +272,12 @@ THREE.TDSLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 				material.shininess = shininess;
 				this.debugMessage( '   Shininess : ' + shininess );
 
+			} else if ( next === MAT_TRANSPARENCY ) {
+
+				var opacity = this.readWord( data );
+				material.opacity = opacity*0.01;
+				this.debugMessage( '  Opacity : ' + opacity );
+                		material.transparent = opacity<100 ? true : false;
```

Comment:
Formatting
```js
var opacity = this.readWord( data );
material.opacity = opacity * 0.01;
this.debugMessage( '  Opacity : ' + opacity );
material.transparent = opacity < 100 ? true : false;
```
Please remember to use tabs, not spaces.

Reply:
@wemamawe Still interested in fixing this formatting issues? Besides, the loader is still missing the declaration of `MAT_TRANSPARENCY  and thus produces a runtime error.

Live demo: https://jsfiddle.net/7ju6Lw28/

Reply:
@Mugen87 I delete that coment befor `var MAT_TRANSPARENCY = 0xA050;`

Reply:
> I delete that coment befor var MAT_TRANSPARENCY = 0xA050;

Have you forgotten to commit this change? I can't see this change in the diff (besides, the live demo is still broken).

Reply:
![image](https://user-images.githubusercontent.com/29365634/65209243-114cb700-daca-11e9-8491-01070a96e5e5.png)
now it works